scripts/deepnano2_caller.py

```diff
@@ -25,29 +25,44 @@ def rescale_signal(signal):
     signal /= mad
     return signal
 
+def add_time_seconds(base_time_str, delta_seconds):
+    base_time = datetime.datetime.strptime(base_time_str, '%Y-%m-%dT%H:%M:%SZ')
+    base_time += datetime.timedelta(seconds=delta_seconds)
+    return base_time.strftime('%Y-%m-%dT%H:%M:%SZ')
+
 def call_file(filename):
     out = []
     try:
         with get_fast5_file(filename, mode="r") as f5:
+            ftype = check_file_type(f5) # single-read/multi-read
             for read in f5.get_reads():
-                read_id = read.get_read_id()
```

Comment:
`get_read_id()` method will soon be deprecated. New way of getting value is introduced.